pkg/executor/table_reader.go

```diff
@@ -138,6 +142,21 @@ func (e *TableReaderExecutor) setDummy() {
 	e.dummy = true
 }
 
+func (e *TableReaderExecutor) memUsage() int64 {
```

Comment:
We should assign a tracker to TableReaderExecutor and attach it to IndexLookup, and TableReader tracks the memory usage itself.

Reply:
https://github.com/pingcap/tidb/blob/c1cae24a87792055e96a213afc9f1459a183b767/pkg/executor/distsql.go#L769-L783

The memTracker of `tableWorker` has been attached to memTracker of `IndexLookUpExecutor `